### ISSUE TITLE
ja4tscan fails to build with latest zmap

### ISSUE THREAD

**Issue by noeltimothy** [FoxIO-LLC/ja4tscan#3]
make install gives the folllowing error:

```
/root/ja4tscan/zmap/src/probe_modules/probe_modules.c:77:6: error: conflicting types for 'fs_add_system_fields'; have 'void(fieldset_t *, int,  int)' {aka 'void(struct fieldset *, int,  int)'}
   77 | void fs_add_system_fields(fieldset_t *fs, int is_repeat, int in_cooldown)
      |      ^~~~~~~~~~~~~~~~~~~~
In file included from /root/ja4tscan/zmap/src/probe_modules/probe_modules.c:19:
/root/ja4tscan/zmap/src/probe_modules/probe_modules.h:117:6: note: previous declaration of 'fs_add_system_fields' with type 'void(fieldset_t *, int,  int,  const struct timespec)' {aka 'void(struct fieldset *, int,  int,  const struct timespec)'}
  117 | void fs_add_system_fields(fieldset_t *fs, int is_repeat, int in_cooldown, const struct timespec ts);
      |      ^~~~~~~~~~~~~~~~~~~~
    
```

**Comment by noeltimothy** [comment]
Furthermore, the the new zmap version is also causing a segmentation fault.

**Comment by noeltimothy** [comment]
Closed, this is fixed on the latest 4.1.0 release
